src/web/src/getlog_GET.ecpp

```diff
@@ -279,238 +280,392 @@ class _clonedcode_Compressor
         void clear()
         {
           _zbody.makeEmpty();
-          /* This is in newer codebase : _deflator.reinitialize(); which does "deflateReset(&_stream)" */
+          // This is in newer codebase : _deflator.reinitialize(); which does "deflateReset(&_stream)"
           _crc = 0;
           _size = 0;
         }
 }; // class _clonedcode_Compressor
 
+// journald logs export to logfile
+int exec_journalctl_command (const std::string& logfile)
+{
+    // 'www-data' user must be owned by 'systemd-journal' group,
+    // to let journalctl access journald files (see fty-core)
+    // usermod -a -G systemd-journal www-data
+    // avoid wrapper usage /usr/libexec/bios/journalctl
```

Comment:
Also avoid using 42ITy wrapper (/usr/libexec/bios/journalctl) by calling the real version from /bin